requirements.in

```diff
@@ -1,17 +1,17 @@
 # Run `make freeze-requirements` to update requirements.txt
 # with package version changes made in requirements-app.txt
 
-Flask==1.1.2
+Flask==2.1.1
 
 gunicorn==20.1.0
```

Comment:
Should we be doing the same thing we do in admin and not pin Eventlet?

https://github.com/alphagov/notifications-admin/blob/3a1ac189ff6800d4789b22e4763e5550719e8802/requirements.in#L22

There’s a CVE out against Eventlet version `0.30.2` https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-21419

Reply:
👍 Now done in [e77ba3b](https://github.com/alphagov/document-download-frontend/pull/118/commits/e77ba3bcb39d1e2f342e8abcb2b9d85e3745aefb)